src/routes/news/components/List.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React from 'react'
+import { FlatList } from 'react-native'
+import styled from 'styled-components/native'
+import {
+  LocalNewsModel,
+  TunewsModel
+} from '@integreat-app/integreat-api-client'
+
+const Loader = styled.ActivityIndicator`
+  margin-top: 7px;
+`
+const keyExtractor = (item, index) => `${index}`
+
+interface ListPropTypes {
```

Comment:
Why interface?

Reply:
I got an un-clear type-script error when I used type instead of an interface  but now it's fixed.